include/sensors/core/fdc1004.hpp

```diff
@@ -70,7 +73,7 @@ constexpr std::size_t CONVERSION_BITS = 24;
 constexpr float MAX_MEASUREMENT_PF = 15;
 constexpr float MAX_RAW_MEASUREMENT =
     float(std::numeric_limits<int32_t>::max() >>
-          ((sizeof(int32_t) * 8) - CONVERSION_BITS));
+          ((sizeof(uint32_t) * 8) - CONVERSION_BITS));
```

Comment:
let's not change this - it makes no difference to the generated code (the value is 4 either way) but everything else is `int32` and it makes things nicer to keep it the same